File: check-if-two-string-arrays-are-equivalent/check-if-two-string-arrays-are-equivalent.cpp
class Solution {
public:
    bool arrayStringsAreEqual(vector<string>& word1, vector<string>& word2) {
        string ans="";
        string res="";
        for(int i=0;i<word1.size();i++){
            ans=ans+word1[i];
        }
        for(int i=0;i<word2.size();i++){
            res=res+word2[i];
        }
        if(ans==res){
            return 1;
        }else{
            return 0;
        }
    }
};

File: 1781-check-if-two-string-arrays-are-equivalent/check-if-two-string-arrays-are-equivalent.java
class Solution {
    public boolean arrayStringsAreEqual(String[] word1, String[] word2) {
        StringBuilder str1 = new StringBuilder();
        StringBuilder str2 = new StringBuilder();

        for (String x : word1) {
            str1.append(x);
        }
        for (String y : word2) {
            str2.append(y);
        }
        return (str1.toString()).equals(str2.toString());

    }
}